lib/client.js

```diff
@@ -123,18 +125,19 @@ Client.prototype._invoke = function(method, arguments, location, callback, optio
         assert.ok(!style || style == 'document', 'invalid message definition for rpc style binding');
         message = self.wsdl.objectToDocumentXML(input.$name, arguments, input.targetNSAlias, input.targetNamespace);
     }
-    xml = "<soap:Envelope " + 
-            "xmlns:soap=\"http://schemas.xmlsoap.org/soap/envelope/\" " +
+    
+    xml = "<"+soapHeader+":Envelope " + 
```

Comment:
Ok, that's actually the XML namespace.
It shouldn't cause issues on the server, you can normally put what you want. What are the errors returned?
But if this change is necessary, can you just change the name from soapHeader to soapNamespace for more consistency?
LGTM otherwise.